0.42.2: parallel build fails

When make is executed with `-j1` everything is fine.
```
+ /usr/bin/make -O -j48 V=1 VERBOSE=1
/usr/bin/make  all-recursive
Making all in dist
make[2]: Nothing to be done for 'all'.
Making all in doc
Making all in man
make[3]: Nothing to be done for 'all'.
make[3]: Nothing to be done for 'all-am'.
Making all in include
make[2]: Nothing to be done for 'all'.
Making all in samples
make[2]: Nothing to be done for 'all'.
Making all in scripts
make[2]: Nothing to be done for 'all'.
Making all in src
Making all in .
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src'
/bin/sh ../build-aux/ylwrap lex.l lex.yy.c lex.c -- flex
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src'
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I.. -I../include    -DSTATIC=static -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -flto=auto -flto-partition=none -Os -Wall -c -o libcgroup_la-log.lo `test -f 'log.c' || echo './'`log.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I.. -I../include -DSTATIC=static -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -flto=auto -flto-partition=none -Os -Wall -c log.c  -fPIC -DPIC -o .libs/libcgroup_la-log.o
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src'
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I.. -I../include    -DSTATIC= -DUNIT_TEST -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -flto=auto -flto-partition=none -Os -Wall -c -o libcgroupfortesting_la-log.lo `test -f 'log.c' || echo './'`log.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I.. -I../include -DSTATIC= -DUNIT_TEST -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -flto=auto -flto-partition=none -Os -Wall -c log.c  -fPIC -DPIC -o .libs/libcgroupfortesting_la-log.o
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src'
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I.. -I../include    -DSTATIC= -DUNIT_TEST -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -flto=auto -flto-partition=none -Os -Wall -c -o libcgroupfortesting_la-wrapper.lo `test -f 'wrapper.c' || echo './'`wrapper.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I.. -I../include -DSTATIC= -DUNIT_TEST -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -flto=auto -flto-partition=none -Os -Wall -c wrapper.c  -fPIC -DPIC -o .libs/libcgroupfortesting_la-wrapper.o
wrapper.c: In function 'cgroup_add_value_string':
wrapper.c:199:50: warning: format '%d' expects argument of type 'int', but argument 3 has type 'long unsigned int' [-Wformat=]
  199 |   fprintf(stderr, "value exceeds the maximum of %d characters\n",
      |                                                 ~^
      |                                                  |
      |                                                  int
      |                                                 %ld
  200 |    sizeof(cntl_value->value) - 1);
      |    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |    |
      |    long unsigned int
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src'
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I.. -I../include    -DSTATIC= -DUNIT_TEST -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -flto=auto -flto-partition=none -Os -Wall -c -o libcgroupfortesting_la-lex.lo `test -f 'lex.c' || echo './'`lex.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I.. -I../include -DSTATIC= -DUNIT_TEST -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -flto=auto -flto-partition=none -Os -Wall -c lex.c  -fPIC -DPIC -o .libs/libcgroupfortesting_la-lex.o
lex.l:18:10: fatal error: parse.h: No such file or directory
   18 | #include "parse.h"
      |          ^~~~~~~~~
compilation terminated.
make[3]: *** [Makefile:634: libcgroupfortesting_la-lex.lo] Error 1
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src'
make[3]: *** Waiting for unfinished jobs....
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src'
/bin/sh ../build-aux/ylwrap parse.y y.tab.c parse.c y.tab.h `echo parse.c | sed -e s/cc$/hh/ -e s/cpp$/hpp/ -e s/cxx$/hxx/ -e s/c++$/h++/ -e s/c$/h/` y.output parse.output -- bison -y -d
/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src/parse.y:48.14-15: warning: POSIX yacc reserves %type to nonterminals [-Wyacc]
   48 | %type <name> ID DEFAULT group_name
      |              ^~
/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src/parse.y:48.17-23: warning: POSIX yacc reserves %type to nonterminals [-Wyacc]
   48 | %type <name> ID DEFAULT group_name
      |                 ^~~~~~~
/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src/parse.y: warning: 4 shift/reduce conflicts [-Wconflicts-sr]
updating parse.h
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src'
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I.. -I../include    -DSTATIC=static -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -flto=auto -flto-partition=none -Os -Wall -c -o libcgroup_la-wrapper.lo `test -f 'wrapper.c' || echo './'`wrapper.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I.. -I../include -DSTATIC=static -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -flto=auto -flto-partition=none -Os -Wall -c wrapper.c  -fPIC -DPIC -o .libs/libcgroup_la-wrapper.o
wrapper.c: In function 'cgroup_add_value_string':
wrapper.c:199:50: warning: format '%d' expects argument of type 'int', but argument 3 has type 'long unsigned int' [-Wformat=]
  199 |   fprintf(stderr, "value exceeds the maximum of %d characters\n",
      |                                                 ~^
      |                                                  |
      |                                                  int
      |                                                 %ld
  200 |    sizeof(cntl_value->value) - 1);
      |    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |    |
      |    long unsigned int
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src'
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I.. -I../include    -DSTATIC=static -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -flto=auto -flto-partition=none -Os -Wall -c -o libcgroup_la-config.lo `test -f 'config.c' || echo './'`config.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I.. -I../include -DSTATIC=static -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -flto=auto -flto-partition=none -Os -Wall -c config.c  -fPIC -DPIC -o .libs/libcgroup_la-config.o
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src'
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I.. -I../include    -DSTATIC= -DUNIT_TEST -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -flto=auto -flto-partition=none -Os -Wall -c -o libcgroupfortesting_la-config.lo `test -f 'config.c' || echo './'`config.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I.. -I../include -DSTATIC= -DUNIT_TEST -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -flto=auto -flto-partition=none -Os -Wall -c config.c  -fPIC -DPIC -o .libs/libcgroupfortesting_la-config.o
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src'
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I.. -I../include    -DSTATIC=static -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -flto=auto -flto-partition=none -Os -Wall -c -o libcgroup_la-lex.lo `test -f 'lex.c' || echo './'`lex.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I.. -I../include -DSTATIC=static -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -flto=auto -flto-partition=none -Os -Wall -c lex.c  -fPIC -DPIC -o .libs/libcgroup_la-lex.o
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src'
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I.. -I../include    -DSTATIC=static -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -flto=auto -flto-partition=none -Os -Wall -c -o libcgroup_la-api.lo `test -f 'api.c' || echo './'`api.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I.. -I../include -DSTATIC=static -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -flto=auto -flto-partition=none -Os -Wall -c api.c  -fPIC -DPIC -o .libs/libcgroup_la-api.o
api.c: In function 'cgroup_change_cgroup_flags':
api.c:3460:37: warning: passing argument 1 of '__xpg_basename' discards 'const' qualifier from pointer target type [-Wdiscarded-qualifiers]
 3460 |      if(procname && strlen(basename(procname))) {
      |                                     ^~~~~~~~
In file included from api.c:48:
/usr/include/libgen.h:34:36: note: expected 'char *' but argument is of type 'const char *'
   34 | extern char *__xpg_basename (char *__path) __THROW;
      |                              ~~~~~~^~~~~~
api.c:3463:17: warning: passing argument 1 of '__xpg_basename' discards 'const' qualifier from pointer target type [-Wdiscarded-qualifiers]
 3463 |        basename(procname));
      |                 ^~~~~~~~
In file included from api.c:48:
/usr/include/libgen.h:34:36: note: expected 'char *' but argument is of type 'const char *'
   34 | extern char *__xpg_basename (char *__path) __THROW;
      |                              ~~~~~~^~~~~~
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src'
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I.. -I../include    -DSTATIC= -DUNIT_TEST -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -flto=auto -flto-partition=none -Os -Wall -c -o libcgroupfortesting_la-api.lo `test -f 'api.c' || echo './'`api.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I.. -I../include -DSTATIC= -DUNIT_TEST -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -flto=auto -flto-partition=none -Os -Wall -c api.c  -fPIC -DPIC -o .libs/libcgroupfortesting_la-api.o
api.c: In function 'cgroup_change_cgroup_flags':
api.c:3460:37: warning: passing argument 1 of '__xpg_basename' discards 'const' qualifier from pointer target type [-Wdiscarded-qualifiers]
 3460 |      if(procname && strlen(basename(procname))) {
      |                                     ^~~~~~~~
In file included from api.c:48:
/usr/include/libgen.h:34:36: note: expected 'char *' but argument is of type 'const char *'
   34 | extern char *__xpg_basename (char *__path) __THROW;
      |                              ~~~~~~^~~~~~
api.c:3463:17: warning: passing argument 1 of '__xpg_basename' discards 'const' qualifier from pointer target type [-Wdiscarded-qualifiers]
 3463 |        basename(procname));
      |                 ^~~~~~~~
In file included from api.c:48:
/usr/include/libgen.h:34:36: note: expected 'char *' but argument is of type 'const char *'
   34 | extern char *__xpg_basename (char *__path) __THROW;
      |                              ~~~~~~^~~~~~
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src'
make[2]: *** [Makefile:683: all-recursive] Error 1
make[1]: *** [Makefile:490: all-recursive] Error 1
```

No it still fails
```
[tkloczko@barrel libcgroup-0.42.2]$ make -O -j48 V=1 VERBOSE=1
make  all-recursive
Making all in dist
make[2]: Nothing to be done for 'all'.
Making all in doc
Making all in man
make[3]: Nothing to be done for 'all'.
make[3]: Nothing to be done for 'all-am'.
Making all in include
make[2]: Nothing to be done for 'all'.
Making all in samples
make[2]: Nothing to be done for 'all'.
Making all in scripts
make[2]: Nothing to be done for 'all'.
Making all in src
Making all in .
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src'
/bin/sh ../build-aux/ylwrap lex.l lex.yy.c lex.c -- flex
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src'
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I.. -I../include    -DSTATIC=static -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -flto=auto -flto-partition=none -Os -Wall -c -o libcgroup_la-log.lo `test -f 'log.c' || echo './'`log.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I.. -I../include -DSTATIC=static -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -flto=auto -flto-partition=none -Os -Wall -c log.c  -fPIC -DPIC -o .libs/libcgroup_la-log.o
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src'
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I.. -I../include    -DSTATIC= -DUNIT_TEST -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -flto=auto -flto-partition=none -Os -Wall -c -o libcgroupfortesting_la-log.lo `test -f 'log.c' || echo './'`log.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I.. -I../include -DSTATIC= -DUNIT_TEST -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -flto=auto -flto-partition=none -Os -Wall -c log.c  -fPIC -DPIC -o .libs/libcgroupfortesting_la-log.o
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src'
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I.. -I../include    -DSTATIC=static -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -flto=auto -flto-partition=none -Os -Wall -c -o libcgroup_la-wrapper.lo `test -f 'wrapper.c' || echo './'`wrapper.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I.. -I../include -DSTATIC=static -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -flto=auto -flto-partition=none -Os -Wall -c wrapper.c  -fPIC -DPIC -o .libs/libcgroup_la-wrapper.o
wrapper.c: In function ‘cgroup_add_value_string’:
wrapper.c:199:50: warning: format ‘%d’ expects argument of type ‘int’, but argument 3 has type ‘long unsigned int’ [-Wformat=]
  199 |   fprintf(stderr, "value exceeds the maximum of %d characters\n",
      |                                                 ~^
      |                                                  |
      |                                                  int
      |                                                 %ld
  200 |    sizeof(cntl_value->value) - 1);
      |    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |    |
      |    long unsigned int
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src'
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I.. -I../include    -DSTATIC= -DUNIT_TEST -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -flto=auto -flto-partition=none -Os -Wall -c -o libcgroupfortesting_la-wrapper.lo `test -f 'wrapper.c' || echo './'`wrapper.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I.. -I../include -DSTATIC= -DUNIT_TEST -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -flto=auto -flto-partition=none -Os -Wall -c wrapper.c  -fPIC -DPIC -o .libs/libcgroupfortesting_la-wrapper.o
wrapper.c: In function ‘cgroup_add_value_string’:
wrapper.c:199:50: warning: format ‘%d’ expects argument of type ‘int’, but argument 3 has type ‘long unsigned int’ [-Wformat=]
  199 |   fprintf(stderr, "value exceeds the maximum of %d characters\n",
      |                                                 ~^
      |                                                  |
      |                                                  int
      |                                                 %ld
  200 |    sizeof(cntl_value->value) - 1);
      |    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |    |
      |    long unsigned int
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src'
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I.. -I../include    -DSTATIC=static -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -flto=auto -flto-partition=none -Os -Wall -c -o libcgroup_la-config.lo `test -f 'config.c' || echo './'`config.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I.. -I../include -DSTATIC=static -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -flto=auto -flto-partition=none -Os -Wall -c config.c  -fPIC -DPIC -o .libs/libcgroup_la-config.o
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src'
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I.. -I../include    -DSTATIC= -DUNIT_TEST -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -flto=auto -flto-partition=none -Os -Wall -c -o libcgroupfortesting_la-config.lo `test -f 'config.c' || echo './'`config.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I.. -I../include -DSTATIC= -DUNIT_TEST -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -flto=auto -flto-partition=none -Os -Wall -c config.c  -fPIC -DPIC -o .libs/libcgroupfortesting_la-config.o
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src'
/bin/sh ../build-aux/ylwrap parse.y y.tab.c parse.c y.tab.h `echo parse.c | sed -e s/cc$/hh/ -e s/cpp$/hpp/ -e s/cxx$/hxx/ -e s/c++$/h++/ -e s/c$/h/` y.output parse.output -- bison -y -d
/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src/parse.y:48.14-15: warning: POSIX yacc reserves %type to nonterminals [-Wyacc]
   48 | %type <name> ID DEFAULT group_name
      |              ^~
/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src/parse.y:48.17-23: warning: POSIX yacc reserves %type to nonterminals [-Wyacc]
   48 | %type <name> ID DEFAULT group_name
      |                 ^~~~~~~
/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src/parse.y: warning: 4 shift/reduce conflicts [-Wconflicts-sr]
updating parse.h
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src'
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I.. -I../include    -DSTATIC=static -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -flto=auto -flto-partition=none -Os -Wall -c -o libcgroup_la-lex.lo `test -f 'lex.c' || echo './'`lex.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I.. -I../include -DSTATIC=static -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -flto=auto -flto-partition=none -Os -Wall -c lex.c  -fPIC -DPIC -o .libs/libcgroup_la-lex.o
lex.l:18:10: fatal error: parse.h: No such file or directory
   18 | #include "parse.h"
      |          ^~~~~~~~~
compilation terminated.
make[3]: *** [Makefile:592: libcgroup_la-lex.lo] Error 1
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src'
make[3]: *** Waiting for unfinished jobs....
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src'
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I.. -I../include    -DSTATIC= -DUNIT_TEST -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -flto=auto -flto-partition=none -Os -Wall -c -o libcgroupfortesting_la-lex.lo `test -f 'lex.c' || echo './'`lex.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I.. -I../include -DSTATIC= -DUNIT_TEST -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -flto=auto -flto-partition=none -Os -Wall -c lex.c  -fPIC -DPIC -o .libs/libcgroupfortesting_la-lex.o
lex.l:18:10: fatal error: parse.h: No such file or directory
   18 | #include "parse.h"
      |          ^~~~~~~~~
compilation terminated.
make[3]: *** [Makefile:634: libcgroupfortesting_la-lex.lo] Error 1
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src'
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I.. -I../include    -DSTATIC=static -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -flto=auto -flto-partition=none -Os -Wall -c -o libcgroup_la-api.lo `test -f 'api.c' || echo './'`api.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I.. -I../include -DSTATIC=static -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -flto=auto -flto-partition=none -Os -Wall -c api.c  -fPIC -DPIC -o .libs/libcgroup_la-api.o
api.c: In function ‘cgroup_change_cgroup_flags’:
api.c:3585:37: warning: passing argument 1 of ‘__xpg_basename’ discards ‘const’ qualifier from pointer target type [-Wdiscarded-qualifiers]
 3585 |      if(procname && strlen(basename(procname))) {
      |                                     ^~~~~~~~
In file included from api.c:48:
/usr/include/libgen.h:34:36: note: expected ‘char *’ but argument is of type ‘const char *’
   34 | extern char *__xpg_basename (char *__path) __THROW;
      |                              ~~~~~~^~~~~~
api.c:3588:17: warning: passing argument 1 of ‘__xpg_basename’ discards ‘const’ qualifier from pointer target type [-Wdiscarded-qualifiers]
 3588 |        basename(procname));
      |                 ^~~~~~~~
In file included from api.c:48:
/usr/include/libgen.h:34:36: note: expected ‘char *’ but argument is of type ‘const char *’
   34 | extern char *__xpg_basename (char *__path) __THROW;
      |                              ~~~~~~^~~~~~
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src'
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I.. -I../include    -DSTATIC= -DUNIT_TEST -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -flto=auto -flto-partition=none -Os -Wall -c -o libcgroupfortesting_la-api.lo `test -f 'api.c' || echo './'`api.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I.. -I../include -DSTATIC= -DUNIT_TEST -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -flto=auto -flto-partition=none -Os -Wall -c api.c  -fPIC -DPIC -o .libs/libcgroupfortesting_la-api.o
api.c: In function ‘cgroup_change_cgroup_flags’:
api.c:3585:37: warning: passing argument 1 of ‘__xpg_basename’ discards ‘const’ qualifier from pointer target type [-Wdiscarded-qualifiers]
 3585 |      if(procname && strlen(basename(procname))) {
      |                                     ^~~~~~~~
In file included from api.c:48:
/usr/include/libgen.h:34:36: note: expected ‘char *’ but argument is of type ‘const char *’
   34 | extern char *__xpg_basename (char *__path) __THROW;
      |                              ~~~~~~^~~~~~
api.c:3588:17: warning: passing argument 1 of ‘__xpg_basename’ discards ‘const’ qualifier from pointer target type [-Wdiscarded-qualifiers]
 3588 |        basename(procname));
      |                 ^~~~~~~~
In file included from api.c:48:
/usr/include/libgen.h:34:36: note: expected ‘char *’ but argument is of type ‘const char *’
   34 | extern char *__xpg_basename (char *__path) __THROW;
      |                              ~~~~~~^~~~~~
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src'
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I.. -I../include    -DSTATIC=static -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -flto=auto -flto-partition=none -Os -Wall -c -o libcgroup_la-parse.lo `test -f 'parse.c' || echo './'`parse.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I.. -I../include -DSTATIC=static -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -flto=auto -flto-partition=none -Os -Wall -c parse.c  -fPIC -DPIC -o .libs/libcgroup_la-parse.o
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src'
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I.. -I../include    -DSTATIC= -DUNIT_TEST -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -flto=auto -flto-partition=none -Os -Wall -c -o libcgroupfortesting_la-parse.lo `test -f 'parse.c' || echo './'`parse.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I.. -I../include -DSTATIC= -DUNIT_TEST -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -flto=auto -flto-partition=none -Os -Wall -c parse.c  -fPIC -DPIC -o .libs/libcgroupfortesting_la-parse.o
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/libcgroup-0.42.2/src'
make[2]: *** [Makefile:683: all-recursive] Error 1
make[1]: *** [Makefile:490: all-recursive] Error 1
make: *** [Makefile:395: all] Error 2
```

--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -1,5 +1,7 @@
 @CODE_COVERAGE_RULES@
 
+BUILT_SOURCES = parse.c parse.h
+
 if WITH_BINDINGS
 BINDINGS_SUBDIR = bindings
 endif

